bittensor/core/dendrite.py

```diff
@@ -811,8 +780,7 @@ async def __aexit__(self, exc_type, exc_value, traceback):
 
         Usage::
 
-            async with bt.dendrite( wallet ) as dendrite:
-                await dendrite.some_async_method()
+            async with bt.dendrite( wallet ) as dendrite: await dendrite.some_async_method()
```

Comment:
Why is this changed?

Reply:
Accidentally. Good catch too! 
Fixed back.